CSOC_Documentation.md

```diff
@@ -61,17 +61,16 @@ A squid proxy is needed for internet connectivity for the admin VM.It is a one t
 `gen3 workon csoc <commons_name>`
 
 A VPC peering connection is used for the communication between the admin VM in the CSOC and the associated data common VPCs.
-1.  A VPC peering request is made when the data commons VPC is spun up (appropriate routes from the data commons VPC to the CSOC VPC via the VPC peering connection  are created in the data commons route tables).`Please note that the “gen3 tfapply” fails when you first run it for the VPC. CSOC admin needs to accept the peering and and then run `gen3 tfplan` and `gen3 tfapply` again.`
+1.  A VPC peering request is made when the data commons VPC is spun up (appropriate routes from the data commons VPC to the CSOC VPC via the VPC peering connection  are created in the data commons route tables). Please note that the `gen3 tfapply` fails when you first run it for the VPC. CSOC admin needs to accept the peering and then run `gen3 tfplan` and `gen3 tfapply` again.
 
 2. VPC peeing acceptance can be done manually or via the script as follows:
-*  Automatic process: We need to login to the csoc_admin VM and run the following command: 
+*  Automatic process: We need to login to the csoc_admin VM and run the following command:
 
 `gen3 approve_vpcpeering_request <child_vpc_name>`
 
 The underneath script basically looks for a new VPC peering request, if there is a request in a pending state, it is accepted, tagged and  an appropriate route to the csoc_main_vpc private subnet route table is added.
 *  Manual Process: Once the VPC Peering connection request is made, the CSOC admin need to do the following (use this process only if `automatic process` explained above give errors):
 	*	Verify the VPC  peering request and accept it. Make sure the VPC peering connection is active. At this time you can give a name to the peering connection for future reference.
 	*	Add a route in the  route table associated to the private subnet - `Destination - Data commons VPC CIDR` & `Target - VPC peering connection`.
-	
-3. Add the CNAME DNS entry in the route 53 of CSOC. Eg. `k8s-{COMMONS-NAME}.internal.io.` pointing to the DNS name of the VPC load balancer for the K8 controller. This is needed to run the `kubectl` commands to the data commons VPC from the admin VM.
 
+3. Add the CNAME DNS entry in the route 53 of CSOC. Eg. `k8s-{COMMONS-NAME}.internal.io.` pointing to the DNS name of the VPC load balancer for the K8 controller. This is needed to run the `kubectl` commands to the data commons VPC from the admin VM.
```

Comment:
this (step 3) is no longer necessary since we moved to using EKS (AWS managed kubernetes)